.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
can't you just run `npx mrm lint-staged` instead of adding all these new files ?
https://prettier.io/docs/en/precommit.html

Reply:
This didnt work anymore for Husky 5+